src/API/SaveFile.hs

```diff
@@ -15,7 +15,7 @@ import Control.Monad.IO.Class
 -- TODO Change SpielResponse to SpielOK/SpielError
 
 handleSaveFile :: SpielFile -> Handler SpielResponses
-handleSaveFile (SpielFile fn contents) = liftIO(do
+handleSaveFile (SpielFile fn contents) = liftIO $ do
     -- TODO needs to check if this file was successfully written, rather than just assuming it is (@montymxb)
     writeFile (fn ++ ".bgl") contents
-    return (SpielResponses [(SpielValue (fn ++ ".bgl" ++ " written successfully"))]))
+    return [(Log (fn ++ ".bgl" ++ " written successfully"))]
```

Comment:
This is actually pretty good, like the idea of standard response for an action with no value.